README.md

```diff
@@ -566,6 +566,62 @@ words from the list by modifying the file.
 Rejected words automatically get flagged by Vale. To modify the list of rejected words, modify the
 [reject.txt](/vale/styles/Vocab/Internal/reject.txt) file.
 
+## Check Formatting
+
+We use [Prettier](https://prettier.io/) to maintain uniform and consistent formatting across the docbase. When you
+commit changes, Prettier formats the staged files automatically. Then, once you create a pull request, it verifies that
+all files comply with our Prettier configuration.
+
+**Note:** The build fails if the Code Formatting check doesn't pass.
```

Comment:
Because this is for the README. We could use GitHub markdown alerts.

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts